config/locales/cy.yml

```diff
@@ -1,215 +1,215 @@
-          attributes:
-            annuity_type:
-              inclusion: is not selected
-            appointment_type:
-              inclusion: is not selected
-        appointment_summary:
-          attributes:
-            appointment_type:
-              inclusion: select a range
-        booking_request_form:
-          attributes:
-            first_name:
-              blank: enter a name
-            last_name:
-              blank: enter a name
-            email:
-              invalid_email: enter a valid email address
-            telephone_number:
-              blank: enter a phone number
-            memorable_word:
-              blank: enter a word
-            date_of_birth:
-              blank: enter a valid date of birth
-            opt_in:
-              accepted: please accept
-            appointment_type:
-              inclusion: must be 50 or over to be eligible for guidance
-            dc_pot:
-              inclusion: select an option
-            where_you_heard:
-              inclusion: select an option
-        complaint:
-          attributes:
-            name:
-              blank: enter a name
-            email_address:
-              invalid_email: enter a valid email address
-        tesco/booking:
-          attributes:
-            first_name:
-              blank: enter a name
-            last_name:
-              blank: enter a name
-            email:
-              invalid_email: enter a valid email address
-            phone:
-              blank: enter a phone number
-              invalid: enter a valid phone number
-            memorable_word:
-              blank: enter a word
-            date_of_birth:
-              blank: enter a valid date of birth
-            dc_pot_confirmed:
-              inclusion: select an option
-            accept_terms_and_conditions:
-              inclusion: please accept
-        telephone_appointment:
-          attributes:
-            first_name:
-              blank: enter a name
-            last_name:
-              blank: enter a name
-            email:
-              invalid_email: enter a valid email address
-            phone:
-              blank: enter a phone number
-            memorable_word:
-              blank: enter a word
-            date_of_birth:
-              blank: enter a valid date of birth
-            dc_pot_confirmed:
-              inclusion: select an option
-            accept_terms_and_conditions:
-              inclusion: please accept
-            where_you_heard:
-              inclusion: select an option
-        feedback_form:
-          attributes:
-            name:
-              blank: enter a name
-            email:
-              invalid: enter a valid email address
-              invalid_email: enter a valid email address
-            message:
-              blank: enter a message
+cy:
+  language_name: Cymraeg
+
+  pension_type_tool:
+    error_title: 'Unable to submit the form'

```

Comment:
@arwelowen Can you also provide the `error_title`, `error` and is the `yes_option` below meant to still be 'Yes'?